notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>I find this code unnecessary long and hard to follow. I'd remove the <code>create_discriminator_block</code> helper, and have a single function. I'd move the comments in an explanatory markdown cell above, and  remove the <code>tf.keras.layers</code> prefix.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Removed! Thanks